### ISSUE TITLE
lapce: blank desktop icon

### ISSUE THREAD

**Issue by hervyqa** [void-linux/void-packages#44987]
### Is this a new report?

Yes

### System Info

Void 5.15.114_1 x86_64 AuthenticAMD notuptodate FFFFFFFF

### Package(s) Affected

lapce-0.2.8_1

### Does a report exist for this bug with the project's home (upstream) and/or another distro?

none

### Expected behaviour

lapce desktop icon appears.

### Actual behaviour

why use actual path instead of icon-name?
https://github.com/void-linux/void-packages/commit/45f5efd7055ecc8ddb36040a95c4c47f620269ea#diff-070924caab2b02270ef5b60a315563d25f8bb12196bd740aa361e80b166d037cR10

```diff
-Icon=dev.lapce.lapce
+Icon=/usr/share/icons/hicolor/scalable/apps/dev.lapce.lapce.svg
```

i'm using kde plasma.

![Screenshot_20230712_105533](https://github.com/void-linux/void-packages/assets/45872139/672a8b01-74bd-4d41-905b-27cf3d6fe203)

how to fix:
rename `Icon=/usr/share/icons/hicolor/scalable/apps/dev.lapce.lapce.svg` to `Icon=dev.lapce.lapce`.

![Screenshot_20230712_111434](https://github.com/void-linux/void-packages/assets/45872139/efd884cb-89fd-477c-9078-1e4ebf7d9cea)

cc maintainer: @atk

### Steps to reproduce

1. `doas xbps-install -S lapce`

**Comment by atk** [comment]
Thanks, I'll have a look.

**Comment by classabbyamp** [comment]
really we should not be vendoring a desktop file, it should be contributed to the upstream.

**Comment by hervyqa** [comment]
> really we should not be vendoring a desktop file

no, not like that I mean. desktop file are [already available](https://github.com/lapce/lapce/blob/master/extra/linux/dev.lapce.lapce.desktop). just need to remove [this patch](https://github.com/void-linux/void-packages/blob/master/srcpkgs/lapce/patches/desktop-icon.patch).
